spec/services/google/scrape_service_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::ScrapeService, type: :service do
+  describe '#call!' do
+    context 'given a VALID keyword_id' do
+      context 'given NO errors' do
+        it 'saves the html of the result page', vcr: 'google/google_no_ads' do
+          keyword = Fabricate :keyword
+
+          described_class.new(keyword_id: keyword.id).call!
+          updated_keyword = keyword.reload
+
+          expect(updated_keyword.result_page_html).to be_a String
+        end
+
+        it 'saves the links count', vcr: 'google/google_no_ads' do
+          keyword = Fabricate :keyword
+
+          described_class.new(keyword_id: keyword.id).call!
+          updated_keyword = keyword.reload
+
+          expect(updated_keyword.links_count).to be_a Integer
```

Comment:
should we test with the actual value, like 1 or 2?

Reply:
If so, should we update the fixture name to reflect that?

from
```
google/google_no_ads
```

to
```
google/google_no_ads_N_links
```

Reply:
Fixed in 511e334d416ff1d2b41e9419cbd7f479575f16f2